src/main/java/org/miracum/etl/fhirtoomop/repository/service/PatientMapperServiceImpl.java

```diff
@@ -28,6 +36,14 @@ public class PatientMapperServiceImpl {
   @Autowired ObservationRepository observationRepository;
   @Autowired CaffeineCacheManager cacheManager;
 
+  @Autowired
+  @Qualifier("goldenMerging")
+  Boolean goldenMerging;
+
+  @Autowired IGenericClient client;
+
+  private Map<String, String> logicalIdToGolden = new ConcurrentHashMap<>();
```

Comment:
This could be a temporary database table.

Reply:
I worry this may flow our heap in the long term?
I wonder if there exists a data structure that has a max size and if it is too large, old entries are deleted?

Reply:
That's another possibility yeah, there are a few options for that

Reply:
Just looking for something simple that we can "get by" with for now.